schema/src/BT01/BT01ProjectInfo.yaml

```
@@ -18,7 +18,7 @@ default_range: string
 classes:
   TerrasosProjectInfo:
```

Oops I need to rename this
```suggestion
  BT01ProjectInfo:
```